File: guess.java

/*
Guess The Number
The most famous simple game written in python
For educational purposes only
*/
import java.util.Scanner;

public class guess {
    public static void main(String[] args) {
        Scanner answer = new Scanner(System.in);

        System.out.println("Guess the number between 1 and 100");

        int number = (int) (Math.random() * 100 + 1);
        int tries = 0;
        int guess;

        while (true) {
            tries += 1;
            System.out.print("nr: ");
            guess = answer.nextInt();
            if (guess == number) {
                break;
            } else if (guess > number) {
                System.out.println("Too high!");
            } else {
                System.out.println("Too low!");
            }
        }

        System.out.println("Congratulations!\nYou guess the number in: " + tries + " tries");
    }
}

File: guess.py
#!/usr/bin/python3
"""
Guess The Number
The most famous simple game written in python
For educational purposes only
"""

import random


def main():
    print("Guess the number between 1 and 100")

    number = random.randint(1, 100)
    tries = 0

    while True:
        tries += 1
        guess = int(input("nr: "))
        if guess == number:
            break
        elif guess > number:
            print("Too high!")
        else:
            print("Too low!")

    print(f"Congratulation!\nYou guess the number in: {tries} tries")


if __name__ == "__main__":
    main()
